tests/test_synth_driver.py

```
@@ -234,6 +235,64 @@ def test_terminate_closes_every_player_and_clears_them(self, driver):
         assert driver._players == {}
 
 
+class TestProcessSpeechSequence:
+    """_process_speech_sequence's own per-task error/cancellation handling
+    (addon/synthDrivers/dengjen_neural_voices/__init__.py)."""
+
+    def test_runs_every_task_in_order(self):
+        ran = []
+
+        def make_task(n):
+            async def task():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `task`

Add return type annotation: `None`

(ANN202)